src/plugins/data/public/ui/query_string_input/language_switcher.tsx

```diff
@@ -53,8 +53,8 @@ interface Props {
 }
 
 export function QueryLanguageSwitcher(props: Props) {
-  const opensearchDashboards = useOpenSearchDashboards();
-  const kueryQuerySyntaxDocs = opensearchDashboards.services.docLinks!.links.query.kueryQuerySyntax;
+  const docLinks = useOpenSearchDashboards().services.docLinks;
+  const osdDQLDocs = docLinks!.links.opensearchDashboards.dql.base;
```

Comment:
Nit: doesn't look like we use `docLinks` anywhere else so I think we can reduce this to one line.

Reply:
will change.